### PR TITLE
Added Spanish translation files of the messages

### LINKED ISSUE #5033: translate user interface and user instructions into English and Spanish
As more and more potential users from English- and Spanish-speaking countries are getting interested in Kitodo, some of them already having started testing or using Kitodo.Production, it would be useful to translate the user interface into both languages.
An English translation of the user interface already exists, except for some details which deserve to be supplemented, such as the user instructions (= short introductory texts in the "help" menu).
As to Spanish, the entire user interface and user instructions would have to be translated.
(effort: low = 2 pt.)

### PR DESCRIPTION
**Pull request by joergleh**
Fixes #5033

### CODE REVIEW

**Comment by henning-gerhardt** [comment]
One question to the current (@Kathrin-Huber) and new (@solth) release management: if Spanish translation is merged must we provide Spanish translation (add or update) changes as well in future pull requests?

**Comment by solth** [comment]
> I will then provide the translations into Spanish once I am able to notice what has been changed.

@joergleh Thank you very much providing the Spanish translation files and for offering to update them when changes are made to the other language files.

I don't know if Github natively provides tools to automatically get informed about changes to specific files (like `messages_de.properties` or `messages_es.properties`, in this case) but there seems to be a third party tool specifically for that purpose: https://app.github-file-watcher.com/
I have never used this "github file watcher" myself so I do not know if it works or is secure, but perhaps someone else has expieriences with that app?

**Review by henning-gerhardt — CHANGES_REQUESTED** [review]
You must at least extend the file [faces-config.xml](https://github.com/kitodo/kitodo-production/blob/master/Kitodo/src/main/webapp/WEB-INF/faces-config.xml) for spanish as without this there is no possibility to switch the language.

I'm surprised that Spanish umlauts must not be translated in some ugly to read encoding like in the German translation, so I don't know if this is working on every system properly.

**Comment by henning-gerhardt** [comment]
> It would suffice if you change the Spanish translation files accordingly to the German and English ones; I will then provide the translations into Spanish once I am able to notice what has been changed.

Thank you for this offering / submission as the main translation too!

File: Kitodo/src/main/resources/messages/messages_es.properties [hunk 1]
```diff
@@ -0,0 +1,1163 @@
...
+confirmDeleteWorkflow=�Borrar el flujo de trabajo "{0}"?
+confirmNewPassword=Confirmar la nueva contrase�a
+confirmRelease=Confirmar el env�o
+confirmUnassignTasks=Todas las tareas en curso asignadas a este usuario se establecen como 'Abiertas'.
+copy=Copiar
+copyRolesToClient=A�adir funciones al cliente
+copyTemplates=Copiar el proyecto incluyendo las plantillas
+correction=Correcci�n
+correctionFor=Correcci�n del paso
+correctionK=K
```
Comment (matthias-ronge) [inline Kitodo/src/main/resources/messages/messages_es.properties:225]: Should be
`correctionK=C`
(first letter of `Corrección`)

**Review by Kathrin-Huber — APPROVED** [review]

**Pull request merged**